app/boards/shields/ergo_s1_oe/ergo_s1_oe.keymap

```diff
@@ -0,0 +1,56 @@
+// | WIN  |  Q   |  W   |  E   |  R   |  T   |                                                   |  Y   |  U   |  I   |  O   |  P   | EQL  |
+// | TAB  |  A   |  S   |  D   |  F   |  G   |                                                   |  H   |  J   |  K   |  L   | SCLN | QUOT |
+// | LCTL |  Z   |  X   |  C   |  V   |  B   |      | CTL  | ALT  |         | CAPS | CTL  |      |  N   |  M   | COMM | DOT  | SLSH | MINS |
+//        | GRV  | BSLS | LEFT | RGHT |      | SHFT |  FN  | HOME |         | PGUP | ENT  | SPC  |      |  UP  |  DN  | LBKT | RBKT |
+//                                                         | END  |         | PGDN |
+                     bindings = <
+        &kp ESC    &kp N1  &kp N2   &kp N3   &kp N4    &kp N5                                                                    &kp N6    &kp N7    &kp N8    &kp N9     &kp N0    &kp BSPC
+        &kp LGUI   &kp Q   &kp W    &kp E    &kp R     &kp T                                                                     &kp Y     &kp U     &kp I     &kp O      &kp P     &kp EQUAL
+        &kp TAB    &kp A   &kp S    &kp D    &kp F     &kp G                                                                     &kp H     &kp J     &kp K     &kp L      &kp SEMI  &kp SQT
+        &kp DEL    &kp Z   &kp X    &kp C    &kp V     &kp B               &kp LCTRL &kp LALT      &kp CAPS  &kp RCTRL           &kp N     &kp M     &kp COMMA &kp PERIOD &kp SLASH &kp MINUS
+                   &kp GRAVE &kp BSLH &kp LEFT &kp RIGHT         &kp LSHFT &mo 1     &kp HOME      &kp PG_UP &kp ENTER &kp SPACE           &kp UP    &kp DOWN  &kp LBKT   &kp RBKT
+                                                                                     &kp END       &kp PG_DN
+                     >;
+              };
+              fn_layer {
+// ---------------------------------------------------------------------------------------------------------------------------------
+// |BT_CLR|  F1  |  F2  |  F3  |  F4  |  F5  |                                                   |      | NLCK |  =   |  /   |  *   |      |
+// |      |  F6  |  F7  |  F8  |  F9  |  F10 |                                                   |      |  7   |  8   |  9   |  -   |      |
+// |      |      |      |      |      |      |                                                   |      |  4   |  5   |  6   |  +   |      |
+// |      |      |      |      |      | PTSC |      |      |      |         |      |      |      |      |  1   |  2   |  3   | ENT  |      |
+//        |      | INS  |      |      |      |      |  FN  |      |         |      |      |      |      |      |      |  .   | ENT  |
+//                                                         |      |         |      |
+                     bindings = <
+        &bt BT_CLR &kp F1  &kp F2   &kp F3   &kp F4    &kp F5                                                                    &trans    &kp KP_NUM&kp EQL   &kp SLASH &kp STAR  &trans
```

Comment:
```suggestion
        &bt BT_CLR &kp F1  &kp F2   &kp F3   &kp F4    &kp F5                                                                    &trans    &kp KP_NUM &kp EQUAL &kp SLASH &kp STAR  &trans
```
Did this compile before with the missing space? Also `EQL` -> `EQUAL` since the former is deprecated.